receiver/splunksearchapireceiver/model.go

```diff
@@ -49,7 +49,8 @@ type List struct {
 
 // SearchResults struct to represent the JSON response from Splunk search results endpoint
 type SearchResults struct {
```

Comment:
I wonder could we link to documentation on these endpoints in comments to at least capture the full set of fields that may be useful?